README.md

```diff
@@ -28,7 +28,7 @@ with appriopriate setters.
 
 ```crystal
 Flux.configure do |settings|
-  settings.host    = ENV["INFLUX_HOST"]? || abort "INFLUX_HOST env var not set"
+  settings.uri     = ENV["INFLUX_URI"]? || abort "INFLUX_URI env var not set"
```

Comment:
Also worth a deprecation not here.

Ditto for the mention in https://github.com/place-labs/flux#configure-the-client.

Reply:
Not sure it is worth it as these env vars are only referenced here in the README and nowhere else.
Might just bump a major.

Reply:
Ah - ignore me completely. When I gave this a glance before I completely missed the fact this _was_ the README.